app/src/main/java/org/fossasia/susi/ai/login/contract/ILoginLogoutModulePresenter.kt

```diff
@@ -0,0 +1,5 @@
+package org.fossasia.susi.ai.login.contract
+
+interface ILoginLogoutModulePresenter {
+    fun loginLogout()
```

Comment:
logout

Reply:
Sir, I didn't understand what you wanted to say by logout. Please explain.

Reply:
Change method name